MapboxNavigation/FeedbackViewController.swift

```diff
@@ -165,10 +178,48 @@ class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecog
         progressBar.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
         progressBar.bottomAnchor.constraint(equalTo: view.safeBottomAnchor).isActive = true
     }
+    
+    func defaultFeedbackHandlers(source: FeedbackSource = .user) -> (send: (FeedbackItem) -> Void, dismiss: () -> Void) {
+        let uuid = eventsManager.recordFeedback()
+        let send = defaultSendFeedbackHandler(uuid: uuid)
+        let dismiss = defaultDismissFeedbackHandler(uuid: uuid)
+        
+        return (send, dismiss)
+    }
+    
+    func defaultSendFeedbackHandler(source: FeedbackSource = .user, uuid: UUID) -> (FeedbackItem) -> Void {
+        return { [weak self] (item) in
+            guard let strongSelf = self else { return }
+            
+            // todo, can this be moved as a delegate on this view controller?
+            //strongSelf.delegate?.mapViewController(strongSelf, didSendFeedbackAssigned: uuid, feedbackType: item.feedbackType)
+            strongSelf.eventsManager.updateFeedback(uuid: uuid, type: item.feedbackType, source: source, description: nil)
+            
+            guard let parent = strongSelf.parent else {
+                strongSelf.dismiss(animated: true)
+                return
+            }
+            
+            strongSelf.dismiss(animated: true) {
+                DialogViewController().present(on: parent)
```

Comment:
For some reason, paren is nil here.

Reply:
It _may_ be because you're using a custom transitioning coordinator. If so, you _may_ need to manually add the child VC and do all the lifecycle methods, such as `UIViewController.willMove(toParentViewController:)`

Reply:
@JThramer solved with https://github.com/mapbox/mapbox-navigation-ios/pull/1605/commits/bc32b5e75b1921a7e7f53bc20f1f11aacf38844d#diff-a25e8b3bfe9ceec368b83a004a6d1708R200. How do you feel about it?

Reply:
Ah yeah. That makes a lot more sense. 👍